coss/club/models.py

```diff
@@ -7,6 +7,41 @@
 from location_field.models.plain import PlainLocationField
 
 
+class ClubRegistration(models.Model):
+    CLUB_TYPE = (
+        ('opensource', 'Open Source'),
```

Comment:
I would prefer to use constants here for these choices.

See: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.Field.choices